stp_core/config.py

```diff
@@ -25,8 +25,8 @@
 
 # Quotas configuration
 ENABLE_DYNAMIC_QUOTAS = False
-MAX_REQUEST_QUEUE_SIZE = 1000
-NODE_TO_NODE_STACK_QUOTA = 100
+MAX_REQUEST_QUEUE_SIZE = 10000
+NODE_TO_NODE_STACK_QUOTA = 1000
```

Comment:
Are we going to increment `NODE_TO_NODE_STACK_SIZE` as well?

Reply:
fixed